README.md

```diff
@@ -135,4 +136,6 @@ If someone has made a pull request and you would like to add code to their branc
   .
   git push <your-fork> <branch-name>
   ```
-  
+## Extension Points Bundler
+
+See the documentation in `packages/extension-points` for how this tooling works.
```

Comment:
I love that this is in a separate package :+1:

Reply:
🎊